index.css

```
@@ -0,0 +1,205 @@
+@import url(/styles/variable.css);
+
+/* ---- 공통 스타일 ---- */
+.line-break {
+  display: block;
+}
+
+.hero {
+  background-color: var(--secondary-color);
+}
+
+.hero-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.hero-dom {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 150px 0;
+  width: 240px;
+}
+
+.hero-header {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 130%;
+  color: var(--primary-text-color);
+}
+
+.hero-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 48px;
+  margin-top: 20px;
+  font-size: 18px;
+  font-weight: 600;
+  color: white;
+  background-color: var(--primary-color);
+  border-radius: 9999px;
+  cursor: pointer;
+}
+
+.hero-img {
+  width: 100%;
+  max-width: 746px;
+}
+
+/* ---- main ---- */
+.main {
+  margin: 0 auto;
+  padding: 24px 0;
+}
+
+.right {
+  text-align: right;
+}
+
+.main-feature {
+  display: flex;
+  flex-direction: column;
+  padding: 28px;
+}
+
+.main-feature img {
+  width: 100%;
+  max-width: 696px;
+  margin: 38px auto;
+}
+
+.feature-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 18px;
+}
+
+.feature-category {
+  font-size: 16px;
+  font-weight: 700;
+  color: var(--primary-color);
+}
+
+.feature-title {
+  font-size: 24px;
+  font-weight: 700;
+  color: var(--primary-text-color);
+  line-height: 130%;
+}
+
+.feature-desc {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 26px;
+  color: var(--primary-text-color);
+}
+
+/* ---- 모바일 (375px 이상) ---- */
+@media (min-width: 375px) {
+  .line-break {
+    display: block;
+  }
+}
+
+/* ---- 태블릿 (768px 이상) ---- */
+@media (min-width: 768px) {
+  .line-break {
+    display: inline;
+  }
+
+  .hero-dom {
+    width: 512px;
+  }
+
+  .hero-header {
+    font-size: 38px;
+  }
+
+  .hero-btn {
+    width: 70%;
+  }
+
+  .main {
+    width: 740px;
+    margin: 40px auto;
+  }
+
+  .main-feature {
+    padding: 20px;
+  }
+
+  .right {
+    text-align: right;
+  }
+
+  .feature-category {
+    font-size: 18px;
+  }
+
+  .feature-title {
+    font-size: 32px;
+  }
+
+  .feature-desc {
+    font-size: 18px;
+    line-height: 26px;
+  }
+}
+
+/* ---- 데스크탑 (1200px 이상) ---- */
+@media (min-width: 1200px) {
```

💊 제안
주석을 다는 것은 좋은 습관입니다. 다만 주석또한 읽어야하는 정보라 중복되지 않는 내용을 입력해주시는 것이 가독성면에서 더 좋아요.
지금 주석중 `1200px 이상`은 아래 css 와 중복되는 내용인 것 같아요~ 

```suggestion
/* ---- 데스크탑 ---- */
@media (min-width: 1200px) {
```